bitcoin/src/pow.rs

```diff
@@ -1077,21 +1077,36 @@ impl kani::Arbitrary for U256 {
     }
 }
 
+/// In test code, U256s are a pain to work with, so we just convert Rust primitives in many places
 #[cfg(test)]
-mod tests {
-    use super::*;
+pub mod test_utils {
```

Comment:
Is there no other way to test these?  It seems adhoc to do this and non-standard to import these tests into block test module.

Reply:
the conversion functions are being used in two places, so it will have to either be duplicated or live in a nested module and imported elsewhere. I prefer the latter